modules/guides/pages/import.adoc

```diff
@@ -0,0 +1,613 @@
+----
+
+[source,nodejs]
+----
+include::nodejs-sdk:howtos:example$import.js[tags=tsvStream,indent=0]
+----
+
+For *JSON* or *JSONL* format, use the xref:https://github.com/uhop/stream-json[stream-json] library.
+
+NOTE: stream-json formats its output with a `{ key: ..., value: ...}` wrapper, so we need to map the stream into the expected format.
+
+[source,nodejs]
+----
+include::nodejs-sdk:howtos:example$import.js[tags=json-jsonl-import,indent=0]
+----
+
+[source,nodejs]
+----
+include::nodejs-sdk:howtos:example$import.js[tags=jsonStream,indent=0]
+----
+
+[source,nodejs]
+----
+include::nodejs-sdk:howtos:example$import.js[tags=jsonlStream,indent=0]
+----
+
+NOTE: {github}
+--
+
+Python::
++
+--
+For *CSV* and *TSV*, use the xref:https://docs.python.org/3/library/csv.html[csv] library:
+
+[source,python]
+----
+include::python-sdk:howtos:example$import.py[tags=csv-tsv-import,indent=0]
+----
+
+[source,python]
+----
+include::python-sdk:howtos:example$import.py[tags=csvImport,indent=0]
+----
+
+[source,python]
+----
+include::python-sdk:howtos:example$import.py[tags=tsvImport,indent=0]
+----
+
+For *JSON* and *JSONL*, use the xref:https://docs.python.org/3/library/json.html[json] library:
+
+[source,python]
+----
+include::python-sdk:howtos:example$import.py[tags=json-jsonl-import,indent=0]
+----
+
+[source,python]
+----
+include::python-sdk:howtos:example$import.py[tags=jsonImport,indent=0]
+----
+
+[source,python]
+----
+include::python-sdk:howtos:example$import.py[tags=jsonlImport,indent=0]
+----
+
+NOTE: {github}
+--
+====
+
+=== Connect to the Couchbase Server
+
+First, you need the connection details for the Couchbase server.
+
+Now decide which bucket and xref:learn:data:scopes-and-collections.adoc[scope and collection] you want to import to (and create them if they don't already exist.)
+
+[{tabs}]
+====
+pass:[.NET]::
++
+--
+[source,csharp]
+----
+include::dotnet-sdk:howtos:example$Import.csx[tags=connect,indent=0]
+----
+
+Learn more about xref:dotnet-sdk:howtos:managing-connections.adoc[].
+
+NOTE: {github}
+--
+Java::
++
+--
+[source,java]
+----
+include::java-sdk:howtos:example$Import.java[tags=connect,indent=0]
+----
+
+If you are using the xref:java:howtos:concurrent-async-apis.adoc#reactive-programming-with-reactor[Reactive API], then use the reactive collection instead:
+
+[source,java]
+----
+include::java-sdk:howtos:example$Import.java[tags=reactiveCollection,indent=0]
+----
+
+Learn more about xref:java-sdk:howtos:managing-connections.adoc[].
+
+NOTE: {github}
+--
+Node.js::
++
+--
+[source,nodejs]
+----
+include::nodejs-sdk:howtos:example$import.js[tags=connect,indent=0]
+----
+Learn more about xref:nodejs-sdk:howtos:managing-connections.adoc[].
+
+NOTE: {github}
+--
+Python::
++
+--
+[source,python]
+----
+include::python-sdk:howtos:example$import.py[tags=connect,indent=0]
+----
+
+Learn more about xref:python-sdk:howtos:managing-connections.adoc[].
+
+NOTE: {github}
+--
+====
+
+=== Inserting the documents
```

Comment:
Either change all the headings to "-ing" forms or change this to "Insert the documents". Also be consistent with casing! Personally I would change all headings to use title case, whereas you have a mixture of title case and sentence case.